docs/.vuepress/components/Builder.vue

```diff
@@ -0,0 +1,237 @@
+<template>
+  <Layout>
+    <template slot="page-top">
+      <div class="builder-container">
+        <h1>🍍 Badge Builder</h1>
+        <p>
+          Use this tool to customize, build and copy a <b>realtime SVG subscriber count badge</b> powered by
+          <a href="https://shields.io/">Shields.io</a> and <b>Substats!</b> 📇 Paste the generated badge directly inside
+          markdown files like your GitHub README to use.
+        </p>
+
+        <dropdown-menu
+          v-model="toggle"
+          transition="translate-fade-down"
+          :interactive="true"
+          style="display: inline-block;"
+        >
+          <div class="key-tip">Source</div>
+          <div class="input-container">
+            <button :class="toggle ? 'dropdown-toggle show' : 'dropdown-toggle'">{{ selectedItem.text }}</button>
+            <div class="dropdown-icon" />
+          </div>
+          <div slot="dropdown">
+            <div class="dropdown-item" @click="select(item)" v-for="item in items" :key="item.id">{{ item.text }}</div>
+          </div>
+        </dropdown-menu>
+
+        <div class="key-tip">Query Key</div>
+        <input class="key-input" type="text" v-model="queryKey" @input="updateBadge" />
+
+        <div class="key-tip">Badge Preview</div>
+        <img :src="badge.image" alt="Shields.io badge" />
```

Comment:
It would be better to add a link here (like https://github.com/kidonng/swiss/blob/cfb5859c6281e007b342e17509b470aa68040982/src/views/substats-badge-creator/index.vue#L18-L20) to match the behavior of rendered markdown.

Some tweaks to the util: https://github.com/kidonng/swiss/commit/cfb5859c6281e007b342e17509b470aa68040982#diff-2a7b3426fa89f69cce05905476752bd0